### DISCUSSION TITLE
Alert

### DISCUSSION THREAD

**Discussion by gmead09** [nVuln/homebridge-lg-thinq#241] [Q&A]
Is there a way to get a push notification or alert when the Washer and or Dryer is done with its cycle?

**Comment by nVuln** [comment]
we have trigger as occupancy sensor, you can create automation on this sensor

**Reply by Realce6** [reply]
Hi @nVuln,

I cannot see this option, only a valve for my washer.
How do I add occupancy sensor?

Thankyou

**Reply by nVuln** [reply]
@Realce6 you need "Enable program finished trigger" in plugin setting
